tools/perform_upgrade.sh

```diff
@@ -20,10 +20,10 @@
 
 set -e
 
-BOARD="$1"
+BOARD="${1}"
```

Comment:
Actually, the Google style guide says to use "$1" here. Those variables are special.

Reply:
Done